config/job_conf.xml.sample_advanced

```diff
@@ -173,25 +173,25 @@
                  pointing of course to the same Kubernetes cluster that you intend to use.
             -->
 
-            <param id="k8s_config_path">/path/to/kubeconfig</param>
+            <!-- <param id="k8s_config_path">/path/to/kubeconfig</param> -->
```

Comment:
Agreed that most parameters are optional, but maybe not this one, as the default (environment variable $KUBECONFIG is rarely set), and hence it will fail if not set. We could add as additional default at the runner to use "~/.kube/config" if you want, to make this param here truly optional by having a sensible default. Let me know @jmchilton